haskell/test/Test/Util.hs

```diff
@@ -86,7 +86,10 @@ originateSegCFMM
   => TokenType -> TokenType -> Storage
   -> m (ContractHandler Parameter Storage)
 originateSegCFMM xTokenType yTokenType storage = do
-  originateSimple "Segmented CFMM" storage $ segCFMMContract xTokenType yTokenType
+  cfmm <- originateSimple "Segmented CFMM" storage $ segCFMMContract xTokenType yTokenType
+  -- Add some slots to the buffers to make the tests more meaningful.
+  call cfmm (Call @"Increase_observation_count") 10
```

Comment:
:thinking: couldn't this be done by modifying the storage instead?

Reply:
Maybe better not perform this in the basic origination function? 
1. This hinders writing tests on observation logic itself (I will have to account for this `+10` even if I wanted to test another scenario)
2. Now mostly all tests will fail if something goes wrong in `Increase_observation_count` logic, probably we'd rather make only few tests be affected by that.

Reply:
Good point, I'll move this away from `originateSegCFMM` then